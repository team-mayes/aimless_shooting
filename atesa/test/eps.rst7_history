eps.rst7_1 S
