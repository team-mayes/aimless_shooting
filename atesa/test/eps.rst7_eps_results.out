0.7000000000000007 1.3000000000000003 1.044353377844
0.7000000000000007 1.3000000000000003 1.0218545299309998
0.7000000000000007 1.3000000000000003 1.185227135742
0.7000000000000007 1.3000000000000003 1.19108807736
0.7000000000000007 1.3000000000000003 1.033953102611
0.7000000000000007 1.3000000000000003 0.936813141482
0.7000000000000007 1.3000000000000003 0.9271406322190001
0.7000000000000007 1.3000000000000003 1.0155567292060002
0.7000000000000007 1.3000000000000003 1.1023455900510002
0.7000000000000007 1.3000000000000003 1.10352148069
